code/datums/diseases/advance/symptoms/chills.dm

```diff
@@ -12,19 +12,20 @@
 	name = "Chills"
 	desc = "The virus inhibits the body's thermoregulation, cooling the body down."
 	illness = "Cold Shoulder"
-	stealth = 0
+	stealth = 1
 	resistance = 2
 	stage_speed = 3
 	transmittable = 2
 	level = 2
 	severity = 2
 	symptom_delay_min = 10
 	symptom_delay_max = 30
-	var/unsafe = FALSE //over the cold threshold
+	symptom_cure = /datum/reagent/medicine/leporazine
 	threshold_descs = list(
 		"Stage Speed 5" = "Increases the intensity of the cooling; the host can fall below safe temperature levels.",
 		"Stage Speed 10" = "Increases the intensity of the cooling even further."
 	)
+	var/unsafe = FALSE //over the cold threshold
```

Comment:
Autodoc as well as tweak the comment do be descriptive to what the var actually does (Determines if this symptom can cause damage)